DEVELOP.rst

```diff
@@ -37,8 +37,6 @@ For help, run:
 
       check   Build, test, and lint the documentation
 
-      delint  Remove any `*.lint` files
```

Comment:
Why remove this?

Reply:
because it's no longer needed. all lint files are kept in a hidden directory, so they don't need to be cleaned up for the user's benefit any more